### ISSUE TITLE
[BUG] Odm-export: ambiguous item values 

### ISSUE THREAD

**Issue by rhe-reliatec** [reliatec-gmbh/LibreClinica#314]
**Description:**
I found in an odm-export a line like this:
```
<ItemData IsNull="Yes" ItemOID="I_22KON_SPIRO_FEF2575" OpenClinica:ReasonForNull="NI" Value="&#228;&#196;6K&#246;rXmKtwQa7S-QSNbBiKUlwgVl681GynlM&#220;3oJBCl&#220;&#246;TvKMTTSTXR&#220;67&#214;G&#246;&#196;B4lo,NI,EI&#196;09TSYTXdh7U2Qg,Rlz&#220;Z?uEsI5liohZ&#252;&#228;Ez&#246;WAI7&#246;Xk4&#246;fI2E9nrA&#252;0?l1G4bQ-TTq?&#220;FJs&#196;vfmqRQCXsEdJURmNvl,&#246;btLKr&#220;7g&#246;EcSui,uMl2KCOxb&#220;RFHTOa&#246;i9dXivY4AnEVzJ-XvyviV5IO">
```
This says, that the item has no value - it is NULL, the user entered NI in the field and it also says the item has a value - the user entered some text.
This is confusing and wrong. Indeed the last state of the field is: it has some value, hence isNull="Yes" is wrong.

**Requirements:**
You can get such ItemData-nodes with LibreClinica 1.0 and with LibreClinica 1.1. I have win10, tomcat9, openjdk 8, postgres13.

The FormDef-nodes in the exported odm-xml show that NullValues is set to NI - what is also shown in the 'Update Study Event Definition'- dialog.
In other words the special treatment of null-values-feature is switched on for the crf.
```
</OpenClinica:FormDetails>
.
Many lines
.
.
<OpenClinica:PresentInEventDefinition PasswordRequired="No" DoubleDataEntry="No" ParticipantForm="No"
IsDefaultVersion="Yes" NullValues="NI," HideCRF="No" AllowAnonymousSubmission="No"
Offline="No" SourceDataVerification="not_required" StudyEventOID="SE_T0BASELINE"></OpenClinica:PresentInEventDefinition>
</OpenClinica:FormDetails>
```
Look at the attribute NullValues. Why a comma?

**Steps to follow:**
1. Prepare a crf with null-values-feature switched on.
2. Enter in a field NI
3. Create a dataset with all items and export as odm: choose *CDISC ODM XML 1.3 Full with OpenClinica extensions*
4. Search your item in the xml and you should find a line like:<br> ```<ItemData ItemOID="I_20STU_SPIRO_FEF2575" IsNull="Yes" OpenClinica:ReasonForNull="NI"/>```
5. Enter in the same field a text which contains **,NI,** eg bcd,NI,qwertz. Save. Export.
6. Search your item in the xml and you will find a line like: <br>```<ItemData ItemOID="I_20STU_SPIRO_FEF2575" IsNull="Yes" OpenClinica:ReasonForNull="NI" Value="bcd,NI,qwertz"> ```

**Expected result:**
I would expect ItemData-nodes without the attributes "IsNull" and "OpenClinica:ReasonForNull" for items with a real value.

**Actual result:**
ItemData-nodes which are marked as NULL-Value which have a value.

**Similar effects:**
Enter:  NI, 
You get: `<ItemData ItemOID="I_20STU_SPIRO_FEF2575" IsNull="Yes" OpenClinica:ReasonForNull="NI,"/>`
Enter: NI, CANNELLONI
You get: `<ItemData ItemOID="I_20STU_SPIRO_FEF2575" IsNull="Yes" OpenClinica:ReasonForNull="NI" Value="NI, CANNELLONI">`

I don't think it's such a good idea to use "magic strings".

**Comment by rhe-reliatec** [comment]
A Customer used it. Maybe it is a good idea to remove this.

**Comment by toskrip** [comment]
I agree. At least from GUI it should be possible to remove this completely so that nobody tries to use it.
